Contest_API.md

```diff
@@ -2096,6 +2102,33 @@ tags below are used.
 | accepted-winner         | A submission was accepted that changed the set of teams currently in the lead.
 | accepted-\<award>       | A submission was accepted that changed the set of teams awarded \<award>. Note that the above 4 are special cases of this.
 
+#### Modifying commentary 
+
+To add a commentary message, clients can use the `POST` method on the commentary endpoint.
+The following [capability](#capabilities) is required to add commentary,
+with description below:
+
+| Name               | Description
+| :----------------- | :----------
+| judge\_comment     | POST a commentary as a judge
+
+All requests must include a valid JSON object with the same properties as the commentary
+endpoint returns from a `GET` request with the following exceptions:
+
+- The `judge\_comment` capability only has access to `POST`. `id`, `time`,
```

Comment:
Nitpick: since no `PUT` is defined, this is not really necessary. But I'm fine with leaving it in, for in case we add that.

Reply:
This was copied from similar capabilities, will try to improve as part of a rewrite.